website/public/css/tasks.styl

```diff
@@ -394,6 +394,11 @@ for $stage in $stages
   a.badge
     position: relative; top:-2px
 
+  a.push-down span
+    -ms-transform: rotate(180deg); /* IE 9 */
+    -webkit-transform: rotate(180deg); /* Chrome, Safari, Opera */
+    transform: rotate(180deg);
+
```

Comment:
This method for changing the icon isn't working on my browser, so we can't rely on it working for all users. It might also produce odd results in future if we change the icon to one that doesn't look meaningful upside down (not likely, but you never know). Did you see crookedneighbor's suggestion in the issue about applying a class to the body when ctrl/cmd is pressed? The class could be named something generic so that it can be reused in cases other than pushToTop/Bottom.